src/main/java/teammates/common/datatransfer/FeedbackQuestionRecipient.java

```diff
@@ -0,0 +1,73 @@
+package teammates.common.datatransfer;
+
+import java.util.Objects;
+
+/**
+ * Represents a recipient of a feedback question.
+ */
+public final class FeedbackQuestionRecipient {
+
+    private String name;
+    private String identifier;
+    private String section;
+    private String team;
+
+    /**
+     * Constructor for FeedbackQuestionRecipient.
```

Comment:
This comment does not value add. I don't think any of the linters enforce this?

Reply:
I think you are right, will update.